.github/PULL_REQUEST_TEMPLATE/run_ci_only.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
This page should probably link to this: https://github.com/firedrakeproject/firedrake/wiki/Running-the-Firedrake-test-suite-with-different-branches-for-subpackages